src/App.css

```diff
@@ -28,6 +28,16 @@
   color: #61dafb;
 }
 
+/* .custom-tab-bar {
```

Comment:
Why are we keeping this if its commented ?

Reply:
It's a special CSS config for the bottom bar that I was testing for IOS. I want the bottom bar size to scale with screen size so that all our other styles in % of screen size make sense. I kept it there to remember :)